projects/igniteui-angular/src/lib/progressbar/progressbar.component.ts

```diff
@@ -670,9 +671,7 @@ export function valueInRange(value: number, max: number, min = 0): number {
     return Math.max(Math.min(value, max), min);
 }
 
-export function toPercent(value: number, max: number) {
-    return Math.floor(100 * value / max);
-}
+export const toPercent = (value: number, max: number) => Math.floor(100 * (value / max));
```

Comment:
```suggestion
export const toPercent = (value: number, max: number) => Math.floor(100 * value / max);
```
This is not needed, as it doesn't matter what precedence multiplication and division are executed with.